### ISSUE TITLE
EyeWitness error

### ISSUE THREAD

**Issue by Chan9390** [reconned/domained#5]
Installation of the tool was successful but I see the following error when I run the tool : 
```
Running Command: python bin/EyeWitness/EyeWitness.py -f output/<domain>-unique.txt --active-scan --no-prompt --headless  -d output/<domain>-09-18-17-04-56-EW 
[*] RDP libraries not found.
[*] Please run the script in the setup directory!


Please provide a domain. Ex. -d example.com
```

**Comment by lianfeng30** [comment]
Mac 和 Windows下不行的
请在kali下安装使用 @Chan9390

**Comment by reconned** [comment]
Thanks lianfeng30! Chan9390, please install the script in Kali Linux / Debian environment and then run "python domained --install" - if you have not already. Let me know if that solves the problem.

**Comment by Chan9390** [comment]
I executed the command `python domained --install`.

**Comment by reconned** [comment]
Issue merged with #4
